cv32e40s/tests/programs/custom/debug_priv_test/debug_priv_test.c

```diff
@@ -168,22 +167,16 @@ int main(void){// TODO: test will failed until issue #278 in core-v-verif/cv32e4
 
 
 
-  // TEST WILL FAIL UNTIL ISSUE #277 (openHW/cv32e40s) IS RESOLVED
-  // TODO: comment the test back in when #277 has been resolved
```

Comment:
Relates to the _"TODO:silabs-robin Comment in when RTL bug is fixed"_ removed above.